packages/ilmomasiina-backend/src/routes/events/getEventDetails.ts

```diff
@@ -99,7 +98,7 @@ export async function eventDetailsForUser(
     ...stringifyDates(event.get({ plain: true })),
     questions,
 
-    quotas: event.quotas!.map((quota) => ({
+    quotas: quotas!.map((quota) => ({
```

Comment:
`!` is probably unnecessary now
```suggestion
    quotas: quotas.map((quota) => ({
```